css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
```

Comment:
**Update**: after talking with the designer, we decided to drop some aspects of the design. The sidebar can now stay on top of the header. The icon doesn't need to animate between open/closed states.